Chapters/Chapter1.md

```diff
@@ -1,5 +1,163 @@
+
+  "I'll be there in a bit."
+
+  She stared at me for a moment - I assume to size up my strange behavior, and then turned to walk back home.
+
+  She reassured me, "I feel it too. There's a chill in the air."
+
+  I felt no comfort.
+
+
+
+*************
+
+
+
+  Kip was at the stove reheating my breakfast and reading a postletter with his morning face on. His hair, the furrowed nest of a bird that took a liking to crooked branches. He looked up, locking his tired eyes to mine, and put on his goofiest morning smile which only complimented his hair. I walked up to nuzzle him. Melna was squatting at the eating table, reading her daily missive from the provincial council.
+
+  "Who's the letter from?", I asked Kip casually.
+
+  "My parents. They're on vacation to the east bay this week."
+
+  "That's a strange place to vacation," said Melna.
+
+  "Yea. They've always been ones to go on adventures to exotic places. They were really wanting to see the sky when Ara is high. Something about brightness meditation."
+
+  I was immediately excited. "Oh, I've heard about that. They say there are no clouds or fog and the whole sky is bright blue like the sunrise. Everything is so hot that you sweat out all your worries."
+
+  "That sounds awful," said Mel, her eyes still focused her work, "Why would someone choose that place to meditate in?"
+
+  Kip shrugged and let out a hopeless hum, more focused on the letter than the now boiling soup pot. I came closer and nudged him away to remove the pot from the focus while he took a moment to snap out of reading.
+
+  "A bowl would be great," I told him.
+
+  "Ah yea. A bowl," he fetched one and held it out for me to pour into.
+
+  Mel walked over to us to check on the food. "Let me make sure it's still like my Amma used to make," she took out a spoon and dipped it into the bowl. "It's too thick and too hot - add some water"
+
+  I did as she said and finally sat down to eat.
+
+  A moment of quiet slurping went by and I could see Mel's mind had moved onto a deeper thought. She released her pen and sat up in her chair, staring longingly out into the sunrise. I could tell she was thinking about traveling again. We had been married too early before we could roam across the world like most single Palinians.
+
+  "Thinking about traveling?" I chimed at her.
+
+  "I am. It's been too long since we've gone anywhere."
+
+  Kip joined, "Last time was when we went to the North coast to live with the Hunnin for a week. I'll never forget how cold that was. Who's idea was that again?" He never missed a chance to remind me.
+
+  "Hey, we all had fun uh... catching fish," I defended myself.
+
+  "You mean having sex," Mel interjected.
+
+  I defended my position, "Hey, catching fish was fun too."
+
+  "There was literally nothing else to do," Kip reminded me.
+
+  "Yea... but I'd totally do it again!"
+
+  And I would. I was kind of a sucker for them. He and Mel shook heads and laughed while I broke the smarmiest smile.
```

Comment:
Something about this line, doesn't sit well with the rest of the narration tone.